deps/http_parser/http_parser.gyp

```diff
@@ -51,13 +51,12 @@
     {
       'target_name': 'http_parser',
       'type': 'static_library',
-      'include_dirs': [ '.' ],
+      'include_dirs': [ '.', 'src' ],
       'direct_dependent_settings': {
-        'defines': [ 'HTTP_PARSER_STRICT=0' ],
         'include_dirs': [ '.' ],
       },
       'defines': [ 'HTTP_PARSER_STRICT=0' ],
-      'sources': [ './http_parser.c', ],
+      'sources': [ 'src/http_parser.c', 'src/api.c', 'src/http.c' ],
       'conditions': [
         ['OS=="win"', {
           'msvs_settings': {
```

Comment:
nit: `msvs_settings` doesn't need to be guarded. GYP ignores it if it's not an Windows build anyway.